pyproject.toml

```diff
@@ -13,5 +13,11 @@ default_section = "THIRDPARTY"
 [tool.coverage.report]
```

Comment:
Shouldn't we have a section
```
[tool.coverage.run]
branch=True
```
in both `pyproject.toml` and `pyproject_no_spark.toml`?

Reply:
So far it didn't seem to be required, I could put it in both of them but maybe redundant.

Reply:
We don't have a `.coveragerc` file in our repo. It looks like you've set this in the _template_ `.coveragerc` file which only applies to projects _created_ by Kedro (i.e when you do `kedro new`) and not the actual kedro repo itself.

Reply:
I had a closer look and applied the suggestion, to both the `.toml` files in the root folder. So both of you are correct, my apologies.

I was under the impression it wasn't needed. Until we started explicitly passing the `.toml` from the `make` commands and the shell script.
On a different note, it's best to have one method of config declaration either via the `.coveragerc` or via the `.toml` - any reasons for both types being in the package? Might be legacy if I'm not wrong.

Note: I took over the PR from Cheuk and so I became the new owner of all the commits here, and this didn't come to mind immediately to me.

Reply:
As far as I know, we do only have `.toml` -- the `.coveragerc` files is within the template that we use to create peoples Kedro packages. It's all a bit meta, but the Kedro package uses `toml` only and contains a cookiecutter template with a `.coveragerc` file as a "static" file that we use to generate a Kedro project.

Reply:
Thanks for explaining, I didn't know the intent behind them.

As we have seen that it can cause assumptions and discussions.